Formula/kvrocks.rb

```diff
@@ -0,0 +1,28 @@
+class Kvrocks < Formula
+  desc "Key-value NoSQL database based on RocksDB and compatible with Redis protocol"
+  homepage "https://github.com/bitleak/kvrocks/"
+  url "https://github.com/bitleak/kvrocks.git",
+    tag:      "v2.0.1",
+    revision: "5f41145adb25a79706a68d233a28a0f633ef255f"
+
+  license "BSD-3-Clause"
+
```

Comment:
```suggestion
```
Or here. (I tried to include this in my other white space suggestion but the GitHub UI doesn't recognise it for some reason.)